platform/core/src/studies/services/wado/studyInstanceHelpers.js

```diff
@@ -257,7 +293,7 @@ async function makeSOPInstance(server, study, instance) {
     instanceNumber: DICOMWeb.getNumber(instance['00200013']),
     imagePositionPatient: DICOMWeb.getString(instance['00200032']),
     imageOrientationPatient: DICOMWeb.getString(instance['00200037']),
-    frameOfReferenceUID: DICOMWeb.getString(instance['00200052']),
+    frameOfReferenceUid: DICOMWeb.getString(instance['00200052']),
```

Comment:
👀 @igoroctaviano, I can never keep track of when/why we caps UID

Reply:
Even in cornerstone, we can see this usage mixed up like frameOfReferenceUID and seriesInstanceUid. Outside OHIF I'm more used to see UID instead of Uid though because this is actually an abbreviation of User ID in which if written using camelcase, it would be UID, upper for each word, right? anyway, it's confusing.

Reply:
WordPress folks having a similar debate: https://github.com/WordPress/gutenberg/issues/6741

Reply:
I'm going to fix this issue for the entire app soonish. We have a source of truth which is the DICOM JSON standard.